index.js

```diff
@@ -1,35 +1,53 @@
 var express = require('express')
 var bodyParser = require('body-parser')
+var DataStore = require('nedb')
 
 const port = 3000
 var BASE_API_PATH = "/api/v1"
+var DB_FILE_NAME = __dirname + "/contacts.json"
 
-var contacts = [
-    {"name": "peter", "phone": "123456789"},
-    {"name": "john", "phone": "987654321"}
-]
 
 console.log("Starting API server...")
 
 const app = express()
 app.use(bodyParser.json())
 
+var db = new DataStore({
+  filename: DB_FILE_NAME,
+  autoload: true
+});
+
 
 app.get('/', (req, res) => {
   res.send('<html><body><h1>My server</h1></body></html>')
 })
 
 app.post(BASE_API_PATH + "/contacts", (req, res) => {
-    console.log(Date() + " - GET /contacts");
-    var contact = req.body;
-    contacts.push(contact);
-    res.sendStatus(201);
+  console.log(Date() + " - POST /contacts");
+    
+  var contact = req.body;
+  db.insert(contact, (err) => {
+      if (err) {
+        console.log(Date() + " - " + err);
+        res.sendStatus(500);
+      }else {
+        res.sendStatus(201);
+      }
+  });
 })
 
 app.get(BASE_API_PATH + "/contacts", (req, res) => {
-    console.log(Date() + " - POST /contacts");
-    res.send(contacts);
-  })
+  console.log(Date() + " - GET /contacts");
+
+  db.find({}, (err, contacts) => {
+    if (err) {
+      console.log(Date() + " - " + err);
+      res.sendStatus(500);
+    }else {
+      res.send(contacts);
```

Comment:
Please, change this code to remove the _id from the information that is send to the user